packages/arm-resource/webpack.config.js

```diff
@@ -0,0 +1,29 @@
+// This is a template webpack config file with minimal configuration.
+// Users are free to create their own webpack configuration files in their application.
+const path = require('path');
+
+/**
+ * @type {import('webpack').Configuration}
+ */
+const config = {
+  mode: 'production',
+  entry: './esm/resource.js',
+  devtool: 'source-map',
+  output: {
+    filename: 'resource.js',
+    path: __dirname,
+    libraryTarget: 'var'
+  },
```

Comment:
@RikkiGibson Can you take a look at this file specifically to make sure it ended up correct? This package is slightly different than the other packages, and I'm hoping that I did the metadata files correctly.

Reply:
Make sure you decide on a name for the library var using the `library` property. See an [example](https://github.com/Azure/ms-rest-js/blob/master/webpack.config.ts#L11) in ms-rest-js.

Reply:
Make sure to update the generated browser examples in README.md.